dagmc_stats.py

```diff
@@ -270,3 +270,50 @@ def get_area_triangle(my_core, meshset, geom_dim):
         area.append(s)
 
     return area
+
+def get_coarseness(my_core,meshset,entity_ranges):
+    """
+    Gets the coarseness of area
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : a meshset containing a certain part of the mesh
+
+    outputs
+    -------
+    coarseness : (list) the coarseness for all surfaces in the meshset
+    """
+    t_p_s = {}
+    coarseness = {}
+    c_data = []
+    for surface in entity_ranges['Surfaces']:
```

Comment:
Maybe we should just pass in `entity_ranges['Surfaces']` instead of the whole `entity_ranges`?